src/common/shorters/bans.ts

```diff
@@ -1,8 +1,46 @@
-import type { APIBan, RESTGetAPIGuildBansQuery } from 'discord-api-types/v10';
+import type {
+	APIBan,
+	RESTGetAPIGuildBansQuery,
+	RESTPostAPIGuildBulkBanJSONBody,
+	RESTPutAPIGuildBanJSONBody,
+} from 'discord-api-types/v10';
 import { BaseShorter } from './base';
 import { GuildBan } from '../../structures/GuildBan';
 
 export class BanShorter extends BaseShorter {
+	/**
+	 * Bulk creates bans in the guild.
+	 * @param guildId The ID of the guild.
+	 * @param body The request body for bulk banning members.
+	 * @param reason The reason for bulk banning members.
+	 */
+	async bulkCreate(guildId: string, body: RESTPostAPIGuildBulkBanJSONBody, reason?: string) {
```

Comment:
This returns an object [with stats bans](https://discord.com/developers/docs/resources/guild#bulk-guild-ban-bulk-ban-response)

Reply:
that's done